### PR TITLE
WoL events, religion scripted triggers, misc bugs

### PR DESCRIPTION
**Pull request by SuccinctScrivener**
A first pass at the WoL events that addresses intra-event issues, but doesn't address inter-event logic or whitespace issues. I also applied the is_*_branch triggers throughout the game and added some new religion related scripted triggers. Let me know if there are any issues. I will try to contribute more to the project in the future.

### CODE REVIEW

File: CleanSlate/common/buildings/00_castleCulture.txt
Comment (schwarherz) [inline CleanSlate/common/buildings/00_castleCulture.txt:45]: As I understand it, this will prevent the building from being able to be constructed, in general, because they no longer meet the potential before the building is constructed.

File: CleanSlate/common/scripted_effects/03_sainthood_effects.txt
Comment (schwarherz) [inline CleanSlate/common/scripted_effects/03_sainthood_effects.txt:22]: I suspect that this change is incorrect. There are six Christian religions unaccounted for in the original that are now caught by the `religion_group` specifically: `cathar`, `lollard`, `waldensian`, `bogomilist`, `nestorian`, and `messalian`

File: CleanSlate/common/scripted_effects/03_sainthood_effects.txt
Comment (schwarherz) [inline CleanSlate/common/scripted_effects/03_sainthood_effects.txt:89]: Similar to the above I believe this may cast too wide of a net.

File: CleanSlate/common/scripted_score_values/04_succession_voting.txt
Comment (schwarherz) [inline CleanSlate/common/scripted_score_values/04_succession_voting.txt:6895]: Not sure the changes to this file are necessary. `has_law` should be valid within the title scope.

File: CleanSlate/events/base_married_life_events.txt
Comment (schwarherz) [inline CleanSlate/events/base_married_life_events.txt:9]: Beware the encoding. CK2 files have to be encoded in `Windows 1252`. With this change, I suspect that all of the files have been changed to `UTF-8` and will need to be changed back

**Review by schwarherz — CHANGES_REQUESTED** [review]
Only major issue I saw was that the files seem to have been converted to `UTF-8` encoding instead of the `Windows 1252` required by CK2. I left comments on a few other minor issues. I also didn't look very in-depth at the events since that's more @Whizzer 's specialty.

**Review by schwarherz — COMMENTED** [review]
We'd appreciate it if you'd join us on our Discord [here](https://discord.gg/yAeeJeA). Just send myself or @Whizzer a message once you've joined and we'll make sure you get access to the dev section so we can discuss everything in more detail with you.

File: CleanSlate/common/scripted_effects/03_sainthood_effects.txt
Comment (schwarherz) [inline CleanSlate/common/scripted_effects/03_sainthood_effects.txt:293]:
```suggestion
# Used by pagans to canonize ancestors. Christians use deciding_upon_saint_effect
```

**Review by schwarherz — CHANGES_REQUESTED** [review]
Only new issue I see is a typo in a comment. Will still wait for @Whizzer 's review before merging

File: CleanSlate/events/wol_business_events.txt
Comment (Whizzer) [inline CleanSlate/events/wol_business_events.txt]: `			preferred_limit = {
				NOT = { religion_group = ROOT }
				is_within_diplo_range = ROOT
			}
			preferred_limit = {
				NOT = { religion = ROOT }
				is_within_diplo_range = ROOT
			}
			preferred_limit = {
				NOT = { religion_group = ROOT }
			}
			preferred_limit = {
				NOT = { religion = ROOT }
			}`

File: CleanSlate/common/scripted_score_values/04_succession_voting.txt
Comment (Whizzer) [reply CleanSlate/common/scripted_score_values/04_succession_voting.txt:6895]: It's valid in title scope, but since it's a realm law, it's only set on a ruler's primary title. Secondary titles may thus not have the same law as the primary title. Checking in the holder's scope checks the primary title of the holder, so that does indeed work properly.

**Review by Whizzer — COMMENTED** [review]
I see a lot of good work here! After a thorough review, I think this is ready for merging. A few general comments: so far, we've kept all the different flag effects and triggers for the various scope types, since they do convey some additional information. Scripted triggers for religion features (incest, excommunication, etc.) should probably be handled with religion flags in the future, but that's probably something to implement across all game files at some point in the future.

**Review by SuccinctScrivener — COMMENTED** [review]
> I see a lot of good work here! After a thorough review, I think this is ready for merging. A few general comments: so far, we've kept all the different flag effects and triggers for the various scope types, since they do convey some additional information. Scripted triggers for religion features (incest, excommunication, etc.) should probably be handled with religion flags in the future, but that's probably something to implement across all game files at some point in the future.

Ah. I had noticed that CleanSlate had truncated *_character_flag to *_flag in a file, it might have been 00_artifact_effects.txt, and had inferred that that was something being done generally. If that's not the case then I'll go ahead and revert the changes shortly.

**Review by schwarherz — APPROVED** [review]

**Pull request merged**
